src/LostAndFound/TimedEvents.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Threading;
+
+namespace LostAndFound
+{
+    public class TimedEvents
+    {
+        public enum WriteSpeed { Normal = 40, Fast = 20, Slow = 80 }
+
+        public static void TimedWrite(string text, WriteSpeed speed = WriteSpeed.Normal)
+        {
+            foreach (char letter in text)
```

Comment:
This will cause an exception if null is passed. We should guard against that.
![image](https://user-images.githubusercontent.com/43797367/85306941-33220280-b47d-11ea-99b0-e31673d1f913.png)

Reply:
What do you suggest?